.github/workflows/flatpak.yaml

```diff
@@ -0,0 +1,23 @@
+on: tag
+
+jobs: 
+  build: flatpack
+  - uses: actions/checkout@v2   # automatically clone the repository
+    runs-on: ubuntu-20.04  # verified on ubuntu-20.04 and should run on any debian distro
+    
+    - name: install Flatpak
+      run: >
+        sudo apt install flatpak gnome-software-plugin-flatpak && flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo && flatpak install flathub org.gnome.Platform//3.36 org.gnome.Sdk//3.36 org.flatpak.Builder
```

Comment:
The two commands for `apt` and `flatpak` should be split on two different lines, there is no need to `&&` them

Reply:
ok, thank you.  that is done now